aries-site/src/examples/components/tag/CreateTagSimple.js

```diff
@@ -0,0 +1,152 @@
+import React, { useContext, useState } from 'react';
+import {
+  Box,
+  Button,
+  Grid,
+  Form,
+  FormField,
+  Select,
+  Tag,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { ThemeContext } from 'styled-components';
+import { simpleTags } from './data';
+import {
+  alphabetize,
+  AppContainer,
+  getRegExp,
+  TagPageHeader,
+  prefix,
+  TagResults,
+  updateCreateOption,
+} from './CreateTag';
+
+const defaultValueOptions = [];
+const defaultValue = '';
+const defaultTags = [];
+
+simpleTags.forEach(datum => {
+  defaultValueOptions.push(datum);
+});
+
+export const CreateTagSimple = () => {
+  const size = useContext(ResponsiveContext);
+  const theme = useContext(ThemeContext);
+
+  // value
+  const [valueOptions, setValueOptions] = useState(
+    alphabetize(defaultValueOptions),
+  );
+  const [value, setValue] = useState(defaultValue);
+  const [valueSearch, setValueSearch] = useState('');
+
+  // tags
+  const [currentTags, setCurrentTags] = useState(defaultTags);
+
+  return (
+    <AppContainer>
+      <TagPageHeader
+        description={`Tags are values that can be assigned to 
+      resources.`}
+      />
+      <Form
+        messages={{ required: 'This is a required field.' }}
+        onSubmit={() => {
+          const exists = currentTags.filter(t => t === value);
+          if (!exists.length) setCurrentTags([...currentTags, value]);
+          setValue(defaultValue);
+        }}
+      >
+        <Grid
+          columns={
+            size !== 'small'
+              ? [['auto', 'medium'], ['auto', 'medium'], 'auto']
+              : '100%'
+          }
+          gap={{ column: 'medium', row: 'small' }}
+        >
+          <FormField
+            name="tag-value-simple"
+            htmlFor="tag-value-simple"
+            label="Value"
+            required={{ indicator: false }}
+          >
+            <Select
+              name="tag-value-simple"
+              id="tag-value-simple"
+              placeholder="Select or create a value"
+              value={value}
+              options={valueOptions}
+              onChange={({ option }) => {
+                if (option.includes(prefix)) {
+                  defaultValueOptions.pop(); // remove Create option
+                  defaultValueOptions.push(valueSearch);
+                  setValueOptions(alphabetize(defaultValueOptions));
+                  setValue(valueSearch);
+                } else {
+                  setValue(option);
+                }
+              }}
+              onSearch={text => {
```

Comment:
Is this double focus correct?

![Screen Shot 2022-01-10 at 2 14 28 PM](https://user-images.githubusercontent.com/1756948/148840748-a5b80d63-9b2a-4364-85b3-7368910bdf67.png)

Reply:
Should the TextInput have placeholder text?

Reply:
Added placeholder text of "Search tag".

Good note on the double focus -- it is an existing behavior given how FormField is handling the focus (wants to display it on itself) and the search field being that actual active element. I have filed a separate issue to resolve this. I believe focus should only be on the TextInput at this point. https://github.com/grommet/hpe-design-system/issues/2200